automate-project.py

```diff
@@ -42,7 +42,7 @@ class bcolors:
 def Blank():
     os.mkdir(projectName)
     os.chdir(projectName)
-    subprocess.run(f"echo # {repoName} >> README.md", shell=True)
```

Comment:
Why are you removing the # from this line? Does it throw an error on mac when it's in there? The reason I put the # in this line is because I want the README name to be a md header.

Reply:
for whatever reason the # in the echo terminates the command and it doesnt create the readme file. ive tried on ubuntu linux and macos. My thinking is best to just remove it.

Reply:
Okay yeah that sounds like the best fix at the moment! Thanks!